alpaca/entities.go

```diff
@@ -362,6 +363,11 @@ type APIError struct {
 }
 
 func APIErrorFromResponse(resp *http.Response) error {
+	if resp == nil {
```

Comment:
This check doesn't make much sense, because if the `resp` is nil, we panic before reaching this. Moreover, according to the documentation of `http.Client.Do` in case of a nil error the response body (and thus the response itself) cannot be nil.